### ISSUE TITLE
The Compound Power Situation...

### ISSUE THREAD

**Issue by sevrick** [dmdorman/hero6e-foundryvtt#1111]
So we can now drag a compound power from a character sheet into a folder that sets up an automatic folder structure.

However the compound powers in the folder seems to have broken the structure of the compound power to no longer act as it should. You can't transfer a compound power equipment from 1 PC to another. Also the new hero designer prefab import doesn't work with the new way of moving compound powers around.

The bottom line is there needs to be a way you can move a compound power around without it being broken up into pieces. Unless there is a button that lets you choose what you want to add back into it.

If this was meant to only be part of the implementation and refinement is coming, then feel free to ignore this post.

Thanks for all your hard work. I nag because I care. :)

**Comment by aeauseth** [comment]
I worked on this today and made several improvements that will be part of the next/3.0.81 release.  I think the drag from actor to actor is fixed.  So are compound powers in a framework.  Not sure about the prefab upload as I don't have a complicated prefab to test with.

**Comment by sevrick** [comment]
Oh crap I feel like an idiot. lol I most of what I said can be ignored. I was look in at the attacks tab when I drug it over. It was adding everything correctly it seems. Even dragging the folder worked. 

The only thing that still needs work is the .hdp compatibility with the new rule.

Sorry my bad. Looks awesome btw now that I know what hell I am doing haha.

**Comment by aeauseth** [comment]
Yup, I confirmed that a combo power within a framework is not loading correctly into the prefab compendium.  I'm looking into that.

**Comment by aeauseth** [comment]
Next release should handle complex prefabs.

![image](https://github.com/dmdorman/hero6e-foundryvtt/assets/13549766/ae3185e4-9180-4b25-8f54-fbd6d5f0c7ef)

![image](https://github.com/dmdorman/hero6e-foundryvtt/assets/13549766/09da7868-3863-428e-9990-a8d08fb956f5)

![image](https://github.com/dmdorman/hero6e-foundryvtt/assets/13549766/e1c509b6-b78c-4df8-a147-433a961d01f3)

**Comment by sevrick** [comment]
You guys are the best keep up the great work.
